CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [8.2]
+
+* **BREAKING CHANGE** `mope build` will run any files matching `main.scss` through PostCSS. Inline loaders should no longer be used on `main.scss`. Expect warnings from PostCSS until we remove unneeded browser prefixes
```

Comment:
might be nice to show a before/after snippet of how the `baseCSSHref` assignment changes for v8